src/main/java/com/github/rodionovsasha/commentservice/controllers/CommentController.java

```diff
@@ -0,0 +1,43 @@
+package com.github.rodionovsasha.commentservice.controllers;
+
+import com.github.rodionovsasha.commentservice.entities.Comment;
+import com.github.rodionovsasha.commentservice.exceptions.CommentNotFoundException;
+import com.github.rodionovsasha.commentservice.services.CommentService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+import static com.github.rodionovsasha.commentservice.Application.API_BASE_URL;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping(API_BASE_URL)
```

Comment:
Every sub mapping starts with `COMMENT_BASE_PATH + `, maybe it makes sense to put the base path to the class level `@RequestMapping` ?